src/shared/dropdown-list-v2/dropdown-list.tsx

```diff
@@ -491,9 +494,10 @@ export const DropdownList = <T, V>({
             return;
         }
 
+        // FIXME: implement onDismiss handling
```

Comment:
based on the prop description, `renderCustomCallToAction` is supposed to pass down a helper function that allows the consumer to manually close the dropdown

currently it just calls `onDismiss` which doesn't actually do anything???